### PR TITLE
fip_60c

### PR DESCRIPTION
**Pull request by Joeysantoro**
This resolves the main final action item of FIP-60 to increase FEI-USDC to 500 AP.
It does not reset FEI-3Crv and FEI-DAI G-UNI as these will be going through the guardian

It also resolves https://linear.app/fei/issue/DEV-2484/fix-broken-fei-pools-in-cr-oracle to fix the unaccounted PCV deposits which need a FEI wrapper.

Lastly, it seeds UMA fuse pool with 1M FEI

### CODE REVIEW

File: test/integration/proposals_config.ts [hunk 1]
```diff
@@ -16,46 +14,23 @@ const proposals: ProposalsConfigMap = {
...
-  fip_64: {
-    deploy: false,
+  // fip_68: {
+  //   deploy: false,
+  //   proposalId: '35352825965290593619963926287136976535992260019512232369033890490308293973890',
+  //   affectedContractSignoff: ['reptbRedeemer', 'fei', 'pegExchanger'],
+  //   deprecatedContractSignoff: [],
+  //   category: ProposalCategory.DAO,
+  //   totalValue: 0,
+  //   proposal: fip_68
```
Comment (ElliotFriedman) [inline test/integration/proposals_config.ts:24]: remove commented fip

**Comment by ElliotFriedman** [comment]
looks like unit and IT are failing and need to be rerun

**Review by eswak — APPROVED** [review]

File: protocol-configuration/dependencies.ts [hunk 2]
```diff
@@ -555,17 +561,38 @@ const dependencies: DependencyMap = {
...
+    contractDependencies: ['collateralizationOracle', 'rariPool91FeiPCVDeposit']
+  },
+  rariPool79FeiPCVDepositWrapper: {
+    contractDependencies: ['collateralizationOracle', 'rariPool79FeiPCVDeposit']
+  },
+  rariPool72FeiPCVDepositWrapper: {
+    contractDependencies: ['collateralizationOracle', 'rariPool72FeiPCVDeposit']
+  },
+  rariPool128FeiPCVDeposit: {
+    contractDependencies: ['rariPool128FeiPCVDepositWrapper']
```
Comment (eswak) [inline protocol-configuration/dependencies.ts:588]: why is this one different ?

Comment (Joeysantoro) [reply protocol-configuration/dependencies.ts:588]: its the deposit not the wrapper

**Review by eswak — APPROVED** [review]

**Pull request merged**